src/components/section/HSection.module.scss

```diff
@@ -0,0 +1,48 @@
+@use 'theme';
+
+.styledSectionWrapper {
+  padding: 8px;
+  margin-bottom: 40px;
+  grid-template-columns: 1fr 2fr;
+}
+
+.styledCollapseIndicator.styledCollapseIndicator {
```

Comment:
Is this double class necessary?

Reply:
Yeah. In the styled component version, we had use `&& {}` to raise its priority. This is how we decided to handle those with css modules.